aws/structure.go

```diff
@@ -1818,7 +1818,9 @@ func sortInterfaceSlice(in []interface{}) []interface{} {
 	a := []string{}
 	b := []interface{}{}
 	for _, v := range in {
-		a = append(a, v.(string))
+		if v != nil {
```

Comment:
Curious - was this important for ECS tasks? Wondering if this has an effect on other types.

Reply:
This caused the AWS provider to crash on Cloudfront distributions, I don't think it ended up affecting ECS Tasks.